app/models/lecture.rb

```diff
@@ -1,5 +1,9 @@
 class Lecture < ApplicationRecord
+  belongs_to :lecturer, class_name: :User
+  has_and_belongs_to_many :participating_students, class_name: :User
+  # has_many :polls, dependent: :destroy
+  enum status: { created: "created", running: "running", ended: "ended" }
```

Comment:
I Like the enum, especially that it is stored as string makes DB readable and usable :+1: